tests/bdd/features/rebuild/log_based_rebuild.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Partial Rebuild
+
+Background:
+  Given io-engine is installed and running
+
+Scenario: Fast rebuild of offlined replica
+  Given a volume with multiple replicas, filled with user data
+  And a replica is taken offline by "OfflineChild" API
+  When the same replica is brought online by "OnlineChild" API
+  Then a log-based rebuild is started after the reconnection
+
+Scenario: Fast rebuild of crashed replica pod
+  Given a volume with multiple replicas, filled with user data
+  When io-engine pod of a replica crashes and restarts
+  Then a log-based rebuild is started after the reconnection
+
+Scenario: Fast rebuild of replica after connection interruption
+  Given a volume with multiple replicas, filled with user data
+  When network connection between nexus and a replica node temporarily goes down and restored
+  Then log-based rebuild is running after reconnection
+
+Scenario: Faulted child is online again within timed-wait period
+  Given a volume with multiple replicas, filled with user data
+  When a child becomes faulted
+  Then the control plane starts timed-wait period
+  And the faulted child is online again within timed-wait period
+  Then a log-based rebuild starts after some time
+
+Scenario: Faulted child is not online again within timed-wait period
+  Given a volume with multiple replicas, filled with user data
+  When a child becomes faulted
+  Then the control plane starts timed-wait period
+  But the faulted child is not online again within timed-wait period
+  Then a full rebuild starts after some time
+
+Scenario: Node goes permanently down while log-based rebuild running
+  Given a volume with multiple replicas, filled with user data
+  When a non-local child becomes faulted
+  And the faulted child is online again within the timeout period
+  Then log-based rebuild is running
+  But the node hosting rebuilding child crashes permanently
+  Then the faulted child is not online again within the timeout period
+  Then a full rebuild starts after some time
+
+Scenario: Volume target is moved before starting log-based rebuild
+  Given a volume with multiple replicas, filled with user data
+  When a child becomes faulted
+  Then the system waits for child to be online again within a timeout period
+  And the faulted child is online again within timeout period
+  When the volume target moves to a different node before the log-based rebuild starts
```

Comment:
Is there a time gap between online child and partial rebuild start?

Reply:
If not, should we reword the bdd to say, `nexus is moved before online child`

Reply:
We are updating this to `the volume target moves to a different node before the log-based rebuild finishes` which will cover both nexus move related scenarios